### PR TITLE
Css structure

### PR DESCRIPTION
**Pull request by mariobrusarosco**
> CSS config

All fonts are loaded in a single '@import'  statement and with all possible 'weihgts'.  Laer on we can decide to use local fonts or to remove the unused 'weights'.

> Others

File are getting 'gziped'

### CODE REVIEW

File: config/webpack.prod.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+const miniCSSExtractPlugin = require('mini-css-extract-plugin')
+const optimizeCSSAssets = require('optimize-css-assets-webpack-plugin')
+const compressionPlugin = require('compression-webpack-plugin')
+const globalVariables = require('./globalVariables')
+const babelConfig = require('./babel-config')
+const stylusConfig = require('./stylus-config')
+
+const prodConfig = env => ({
+  entry: {
+    // main: ["./src/main.js"]
```
Comment (vagfsantos) [inline config/webpack.prod.js]: :scissors:

File: config/stylus-config.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      options: {
+        modules: true,
+        localIdentName: '[name]__[hash:base64:8]',
+        importLoaders: 1
+      },
+    },
+    'postcss-loader',
+    { 
+      loader: 'stylus-loader', 
+      // options: {
```
Comment (vagfsantos) [inline config/stylus-config.js]: :scissors:

File: src/components/App/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import Header from '../Header'
+
+import ted from '../../images/ted.png'
+
+class App extends React.Component {
+
+  render() {
+    return (
+      <>
```
Comment (vagfsantos) [inline src/components/App/index.js:12]: que isso? :thinking:

**Review by vagfsantos — COMMENTED** [review]
WHAAAAAT A BUILD :+1: :1st_place_medal: :clap: :clap: :clap: :clap: :clap: :clap:

File: src/components/App/index.js (hunk 3 above)
Comment (mariobrusarosco) [reply src/components/App/index.js:12]: @vagfsantos Pra nao usar uma "div" nem o "Fragment"  ... É galhofa do Babel 7...

Tem o Elvis operator funcionando já!

File: config/stylus-config.js (hunk 2 above)
Comment (mariobrusarosco) [reply config/stylus-config.js]: This is related to the issue  #1. 
 It'Is gonna be temporary removed on v.1.1 (https://github.com/mariobrusarosco/webpack-boilerplate/releases/tag/v1.1.0) while the issue is active!

Comment (mariobrusarosco) [reply config/stylus-config.js]: This is related to Issue #1. It's gonna be temporary removed on  Build's [v1.1.0](https://github.com/mariobrusarosco/webpack-boilerplate/releases/tag/v1.1.0)

File: config/webpack.prod.js (hunk 1 above)
Comment (mariobrusarosco) [reply config/webpack.prod.js]: Yes, I'll remove! But, i'm not so sure that we can already get rid of the Babel's polyfill yet or not!!!!

I'll take a look!!

**Comment by mariobrusarosco** [comment]
@vagfsantos Updated

**Review by vagfsantos — APPROVED** [review]
QUE FRONT!

**Pull request merged**